jdaviz/configs/mosviz/tests/test_data_loading.py

```diff
@@ -0,0 +1,220 @@
+# Tests data loading in the Mosviz Jdaviz configuration
+
+import pathlib
+from zipfile import ZipFile
+
+from astropy.nddata import CCDData
+from astropy.utils.data import download_file
+import numpy as np
+import pytest
+from specutils import Spectrum1D
+
+
+def test_load_spectrum1d(mosviz_helper, spectrum1d):
+    label = "Test 1D Spectrum"
+    mosviz_helper.load_1d_spectra(spectrum1d, data_labels=label)
+
+    assert len(mosviz_helper.app.data_collection) == 2
+    assert mosviz_helper.app.data_collection[0].label == label
+
+    table = mosviz_helper.app.get_viewer('table-viewer')
+    table.widget_table.vue_on_row_clicked(0)
```

Comment:
this probably shouldn't be needed (may be related to @camipacifici's comment - so having in the test might just help to ensure everything is set correct).  There is also a higher API for this now (`table.select_row(0)`), but that is essentially just a wrapper around this.

(I also realize these were here before and you just moved them - so making changes in likely out of scope - I just wanted to leave a mention somewhere in case it helps us when debugging the spectrum showing on load issue).